### DISCUSSION TITLE
关于macOS的两个问题

### DISCUSSION THREAD

**Discussion by SfYangYang** [AlistGo/alist#4449] [Q&A]
1. 安装好Alist后，我发现只要MacBook息屏，就无法使用了，难道macbook必须一直屏幕常亮才可以吗？
2. 怎样设置为开机自动运行呢，不用每次都输入命令？

**Comment by anwen-anyi** [comment]
> MacBook息屏，就无法使用了

估计是你设置的息屏断网，之前我的Windows笔记本也是息屏自动断网
